### PR TITLE
[WIP] Small fixes

### PR DESCRIPTION
**Pull request by VinGarcia**
I found some typos and a missing step on one of the documents.

I might be wrong about some of them, please don't hesitate to tell me if this is the case.

I should add more changes until the end of the day.

### CODE REVIEW

File: 02-CreatingDockerImage/README.md [hunk 1]
```diff
@@ -224,7 +224,7 @@ To login to ECR, copy this output and execute it as a linux command. The output
```
Comment (VinGarcia) [inline 02-CreatingDockerImage/README.md:224]: Na nota acima menciona-se que o comando `aws ecr get-login --region-us-east-1 --no-include-email` iria fazer o shell executar o output do comando.

Mas me parece que ele não faria isso, será que está faltando algo como um `bash -c "$(...)"`?

**Review by VinGarcia — COMMENTED** [review]
Tem um comando que é referenciado em uma nota que eu acho que está errada, mas não sei bem qual seria o melhor jeito para corrigir.

**Comment by VinGarcia** [comment]
I found a problem in the tutorial "04-DeployEcsCluster" in the very end of section 5.

In the part after we configure the "container to load balancer" section on the Service Creation wizard, we are asked to set the "Path pattern" to `/app*` and then instructed to click on Next Step with the following sentence:

- "If the values look correct, click Next Step."

However, I think we should set the VPC to MyVPC on a section a little down on the page. I did this part of the tutorial twice because the first time it was unable to create the service, but it worked on the second time after I set the VPC to MyVPC.

> Note: I am not sure about the name of the input where I selected `MyVPC`, but it was just little below the part where the health check path was set to `/app`. I can't go back there to see it right now because I have already deleted the cluster and everything else so I can have a fresh start tomorrow or after.

I am not confident about adding this change to this PR because the website did not work after that and I don't know what I did wrong. But the fact is I was unable to create the service by following the original instructions.

**Comment by bemer** [comment]
We just updated the workshop, so these errors are not present anymore.

**Pull request closed without merging**